src/main/java/org/opensearch/flowframework/exception/FlowFrameworkException.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.exception;
+
+import javax.ws.rs.core.Response;
+
+/**
+ * Representation of Flow Framework Exceptions
+ */
+public class FlowFrameworkException extends RuntimeException {
```

Comment:
By inheritance this class implements `Serializable` so it needs a `serialVersionUID`.
```suggestion
public class FlowFrameworkException extends RuntimeException {

    private static final long serialVersionUID = 1L;
```